### ISSUE TITLE
Netzwerkspiel auswählen und aktivieren

### ISSUE THREAD

**Issue by TimoKramer** [TimoKramer/PatFra#27]
Wenn der Spieler im Netzwerk-Menü ist,
wenn ein Netzwerk-Spiel verfügbar ist,
wenn der Spieler auf 'Bereit' klickt,
dann startet ein Netzwerk-Spiel.

**Comment by TimoKramer** [comment]
sehr optional, würde ich vorschlagen...nice to have

**Comment by freudenb** [comment]
Wenn wir nen Multiplayer Modus einbauen wollen müssten wir das glaube ich schon von Anfang an planen und auch den Rest darauf auslegen. Ich würde ihn eher weglassen oder dann am Ende schauen ob sich einfache Methode anbietet den noch einzubauen, muss mann dann schauen ob sich der Aufwand lohnt.

**Comment by TimoKramer** [comment]
ja, das sehe ich genauso. Ich denke, wir können diesen Punkt aber drin lassen, wegen der Vollständigkeit und eben dann als 'kann'-Requirement. D.h. wenn wir Lust und Zeit drauf haben, machen wir das noch, aber eben nicht als 'muss'- oder 'soll'-Req, sondern 'kann'

OK?

**Comment by freudenb** [comment]
ja ok machen wir so
